### PR TITLE
Introduced image editing to the JS API

### PR DESCRIPTION
**Pull request by simba909**
**Note**: This PR depends on another one I made on the AIQCoreLib repo: https://github.com/appear/AIQCoreLib/pull/1

This adds a new JS API endpoint for graphically editing images:

``` javascript
aiq.imaging.edit(imagePath, callbacks);
```

where `imagePath` is the full path to an image on the device and `callbacks` is an object containing callbacks:

``` javascript
var callbacks = {
  success: function (imageData) {
    // This property has to be here, and it has to be callable
    // imageData has two properties:
    var contentType = imageData.contentType;
    var editedImagePath = imageData.resourceUrl;
  },
  failure: function () {
    // This is an optional property
  }
};
```

**Note**: the original image will not be altered. A new image will be created based on the user's edits and saved to a temporary directory on the device.

### CODE REVIEW

**Comment by viart** [comment]
see nothing fishy here

File: Pod/Assets/plugins/aiq-plugin-imaging/www/imaging.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ cordova.define('aiq-plugin-imaging', function(require, exports, module) {
         // Empty constructor
     }
     
-    Imaging.prototype.capture = function(idOrSettings, settingsOrNil) {
+    Imaging.prototype.capture = function (idOrSettings, settingsOrNil) {
         console.warn('aiq.imaging.capture() is deprecated, use navigator.camera.getPicture() isntead');
```
Comment (mansn) [inline Pod/Assets/plugins/aiq-plugin-imaging/www/imaging.js:10]: It's not really part of this PR. But there's a typo in the last word here.. "isntead". =)

**Comment by mansn** [comment]
lgtm
